maven-surefire-plugin/src/site/apt/examples/toolchains.apt.vm

```diff
@@ -54,3 +54,36 @@ Using Maven Toolchains with ${thisPlugin}.
 +---+
 
          The above example assumes that your toolchains.xml contains a valid entry with these values.
+
+* Minimal Java Versoin for Toolchains
```

Comment:
Version

Reply:
Sorry I copy/paste you text just above :point_up: https://github.com/apache/maven-surefire/pull/701#discussion_r1425396777, I will fix it.

Reply:
By bad